python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -722,19 +908,60 @@ async def arun(self, checkpoint=False):
                 self.log.info(f"Skipping verification for {im_type}. ")
                 response_ocps_verify_pipetask = response_ocps_calib_pipetask
                 previous_step = "generation"
-            # TODO: Currently, we are auto-certifying the calibration.
-            # However, we will have to add a check that looks
-            # at the output of cp_verify in step 3) and does not certify the
-            # calibration if cp_verify does not pass: DM-31897
-            # 4. Certify the calibration
+
+            # 4. Certify the calibration if the verification tests passed
             if not response_ocps_verify_pipetask["phase"] == "completed":
                 raise RuntimeError(
                     f"{im_type} {previous_step} not completed successfully: "
                     f"Status: {response_ocps_verify_pipetask['phase']}. "
                     f"{im_type} certification could not be performed."
                 )
             else:
-                await self.certify_calib(im_type, job_id_calib)
+                # Check the verification statistics and decide whether
+                # the master calibration gets certified or not.
+                job_id_verify = response_ocps_verify_pipetask["jobId"]
+
+                # Check verifycation tests only if bias,
```

Comment:
I am a bit confused about this statement. Aren't this all the image types this script operates with? Seems a bit redundant to have the if/else statement. But maybe I am missing something.

Reply:
`im_type` can be an image or calibration type in `["BIAS", "DARK", "FLAT", "DEFECTS", "PTC"]`, but verification is not yet implemented for `"DEFECTS"` and `"FLAT"`. I added this clarification to the comment.